githubteacher-fuquay.md

```diff
@@ -0,0 +1,5 @@
+## Fuquay Varina, NC
+
+### Places to eat
```

Comment:
Hey @RoyChapman since you can comment here you should be able to open an issue! Can you please do that and @brianamarie and @halfaro1 will help!